src/content/content_manager.cc

```diff
@@ -406,7 +406,7 @@ void ContentManager::addVirtualItem(const std::shared_ptr<CdsObject>& obj, bool
 
     std::error_code ec;
     auto dirEnt = fs::directory_entry(path, ec);
-    if (!dirEnt.is_regular_file(ec))
+    if (ec.value() || !dirEnt.is_regular_file(ec))
```

Comment:
You can just check `ec` as it has a boolean overload for non-zero :+1:

Reply:
Thanks!